__tests__/commands/_helpers.js

```diff
@@ -81,9 +81,21 @@ export async function run<T, R>(
 
   let cwd;
   if (fixturesLoc) {
-    const dir = path.join(fixturesLoc, name);
+    let parentDir = name;
+    let childDir = '';
+
+    // if the name is a path, copy the parent dir, then use the child dir as the cwd
+    // there's some existing tests which reference fixtures in a parent dir (../), so excluding those
+    const nameSepIndex = name.indexOf('/');
+    if (nameSepIndex !== -1 && name.indexOf('../') === -1) {
```

Comment:
This is hard to parse for me.
What if there are multiple separators in the name a/b/c?
Also it gets complicated with relative paths to the fixture.
Any idea to simplify this?
Maybe move the logic for changing CWD to the test itself?